core/src/main/java/io/paradoxical/cassieq/dataAccess/MonotonicRepoImpl.java

```diff
@@ -43,6 +43,15 @@ public MonotonicRepoImpl(Session session, @Assisted QueueName queueName) {
         return current == null ? MonotonicIndex.valueOf(0) : current;
     }
 
+    @Override
+    public void deleteAll() {
```

Comment:
eh. given that the repo is queue specific this method is misleading.... no good name ideas thouhg